hackathon_site/dashboard/frontend/src/components/teamDetail/TeamActionTable/TeamActionTable.tsx

```diff
@@ -0,0 +1,97 @@
+import {
+    Grid,
+    Paper,
+    Table,
+    TableBody,
+    TableCell,
+    TableContainer,
+    TableRow,
+    Button,
+} from "@material-ui/core";
+import Typography from "@material-ui/core/Typography";
+import React from "react";
+import NotificationImportantIcon from "@material-ui/icons/NotificationImportant";
+import CallSplitIcon from "@material-ui/icons/CallSplit";
+import MergeTypeIcon from "@material-ui/icons/MergeType";
+import DeleteIcon from "@material-ui/icons/Delete";
+
+import styles from "./TeamActionsTable.module.scss";
+
+const TeamActionTable = () => {
+    return (
+        <Grid container spacing={1} direction="column" item md={6} xs={12}>
+            <Grid item>
+                <Typography variant="h2">Actions</Typography>
+            </Grid>
+            <TableContainer component={Paper}>
+                <Table
+                    size="small"
+                    style={{
+                        minWidth: 360,
+                        height: 256.8,
+                    }}
+                >
+                    <TableBody>
+                        <TableRow>
+                            <TableCell className={styles.actionItem}>
```

Comment:
I have a code suggestion for here
You see how every <TableRow> has a lot of repeated code? What you could do is create an array that holds all the unique information to that row e.g.

```
// before return statement
const actionButtons = [
   {
      icon: <NotificationImportantIcon />,
      text: "Notify team to come to table to resolve issue",
      onClick: () => {//whatever your function is}
   },
   etc.
]
// in return jsx
<TableBody>
   {actionButtons.map((action) => (
      <TableRow>
          <TableCell ...>
             <Button .... startIcon={action.icon} onClick={action.onClick}> {action.text} </Button>
          </TableCell>
      </TableRow>
   )}
</TableBody>
```
This way you shorten your code *a lot* and it's more readable and easier to change.

This is using your Table code for clarity but do try using the MUI List component instead :)

Reply:
ok, ill take the data out of the return statement like you said